### PR TITLE
feat(rust): improve on text and outputs of enroll, influx and kafka commands

### PR DESCRIPTION
**Pull request by caoakleyii**
## Proposed changes

```sh
❯ o enroll

        .8888888.           dP
       d8'     `8b          88
       88       88 .d8888b. 88  .dP  .d8888b. 88d8b.d8b.
       88       88 88'  `"" 88888"   88'  `88 88'`88'`88
       Y8.     .8P 88.  ... 88  `8b. 88.  .88 88  88  88
        `888888P'  `88888P' dP   `YP `88888P8 dP  dP  dP


       Enrolling your default Ockam identity with Ockam Orchestrator ...

       There is no identity, on this machine, marked as your default.
       Creating a new Ockam identity for you...
     ✔︎ Created identity P64d69d526a9517ae2c2d165a6dd3a57a127d17085ebcbb997a83782d97b33645
       Marked this new identity as your default, on this machine.

       To enroll we need to associate yyour Ockam identifier with an Orchestrator account

     │ First copy this one-time code:  WZXK-GGMX 
     │ Then press  ENTER ↵  to open https://account.ockam.io/activate in your browser.
       
     │ Opening https://account.ockam.io/activate, in your browser, to begin authentication...
     │ Authenticated

       Getting available spaces in your account...
       Found space 52293cb6.
     ✔︎ Marked this space as your default space, on this machine.

       Getting available projects in space 52293cb6...
       Found project default.
     ✔︎ Marked this project as your default project, on this machine.

     ✔︎ Enrolled P64d69d526a9517ae2c2d165a6dd3a57a127d17085ebcbb997a83782d97b33645 as one of the Ockam identities within the Orchestrator space 52293cb6
```

```sh
❯ o tcp-outlet create --to 8086
       Creating TCP Outlet to 127.0.0.1:8086

       There is no node, on this machine, marked as your default.
       Creating a new Ockam node for you...
       Created a new node named default
       Marked this node as your default, on this machine.

     ✔︎ Created a new TCP Outlet on node default from address /service/outlet to 127.0.0.1:8086
❯ o relay create
       Creating Relay...

     ✔︎ Now relaying messages from /project/default/service/8f23232b7ec5c7d7685430e8e30a8fdb → /node/default/service/forward_to_default
❯ o tcp-inlet create
       Creating TCP Inlet at 127.0.0.1:58263...

     ✔︎ TCP Inlet 127.0.0.1:58263 on node default is now sending traffic
       to the outlet at /project/default/service/forward_to_default/secure/api/service/outlet
❯ o kafka-consumer create
       Creating KafkaConsumer service...

     ✔︎ KafkaConsumer service started at 127.0.0.1:4000
       Brokers port range set to 4001-4100
❯ o kafka-producer create
       Creating KafkaProducer service...

     ✔︎ KafkaProducer service started at 127.0.0.1:5000
       Brokers port range set to 5001-5100
```

## Checks

- [ ] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [ ] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [ ] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

**Comment by caoakleyii** [comment]
@mrinalwadhwa  In case you wanted to do any extra testing on this branch or view the output.

File: implementations/rust/ockam/ockam_command/src/node/mod.rs [hunk 1]
```diff
@@ -123,13 +123,19 @@ pub fn spawn_default_node(node_name: &str) -> String {
     create_command.run(quiet_opts);
 
     if let Ok(mut logs) = PARSER_LOGS.lock() {
-        logs.push(fmt_log!("No default node was found."));
         logs.push(fmt_log!(
-            "Created default node, {}",
+            "There is no node, on this machine, marked as your default."
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/node/mod.rs:119]: Typo in this comma here? `no node, on this`

Comment (caoakleyii) [reply implementations/rust/ockam/ockam_command/src/node/mod.rs:119]: I believe the comma is correct here?

"There is no node marked as your default."

", on this machine,"

Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/src/node/mod.rs:119]: OK I see now, I'd probably remove both commas now that I re-read it, but we can leave it as it is!

**Review by adrianbenavides — APPROVED** [review]

**Pull request merged**